src/main/java/org/opensearch/security/compliance/ComplianceConfig.java

```diff
@@ -508,7 +508,8 @@ public boolean writeHistoryEnabledForIndex(String index) {
             return false;
         }
         // if security index (internal index) check if internal config logging is enabled
-        if (securityIndex.equals(index)) {
+        // TODO: Add support for custom api token index?
+        if (securityIndex.equals(index) || securityIndex.equals(ConfigConstants.OPENSEARCH_API_TOKENS_INDEX)) {
```

Comment:
this should be replaced by a matcher as done in AbstractAuditLog class.

Reply:
Sure done in: [58cf8ca](https://github.com/opensearch-project/security/pull/4921/commits/58cf8cac4696ee0e0c8ea6eb58eea10f1fe7a460)